src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -48,19 +53,40 @@ class AssetsTable extends React.Component {
     });
   }
 
-  addDeleteButton() {
-    const newAssetsList = this.props.assetsList.map((asset) => {
+  getNextFocusElementOnDelete(assetToDelete) {
+    let rowToDelete = assetToDelete.row_key;
+    let focusAsset = assetToDelete;
+    let focusRowButton = this.trashcanRefs[assetToDelete.id];
+
+    if (this.props.assetsStatus.text === AssetsConstants.SUCCESS_FILE_DELETE) {
+      if (rowToDelete > 0) {
+        rowToDelete -= 1;
+      }
+
+      focusAsset = this.state.displayAssetsList.find(asset => asset.row_key === (rowToDelete));
+      focusRowButton = this.trashcanRefs[focusAsset.id];
+    }
+
+    return focusRowButton;
+  }
+
+  addSupplementalTableElements() {
```

Comment:
Renamed this fuction as it now has more use cases than just adding the delete button -- will add lock button and now adds an ID for the row # that the data is on.  That row # is used for determining the next focusable element once the status alert is closed.